features/facebookImport/src/styles.css

```diff
@@ -53,11 +53,31 @@
     src: url("../fonts/jost_regular.ttf");
 }
 
+@font-face {
+    font-family: "Jost Book";
+    src: url("../fonts/jost_book.ttf");
```

Comment:
I don't think this has been uploaded... or is used anywhere, is that correct?

Reply:
Yes, true, I will take that out for now and not install it. It seems to be the same as our default Jost* font-weight: 400